src/JobPanel/backend/service_base.py

```diff
@@ -214,24 +218,61 @@ def _process_requests(self):
                 data = request['data']
             assert( 'action' in request.keys() )
             answer = self.call_action(request['action'], data)
```

Comment:
Pokud bude "call_action" provádět LongRequest ve vlákně a pak třeba samo volat self._repeater.send_answer(request_data.id, answer) nebude znát request_data.id, má se teda do "call_action" předávat i to "id"?